src/main/java/it/pagopa/interop/probing/eservice/operations/annotations/validator/StringArrayValidator.java

```diff
@@ -0,0 +1,29 @@
+package it.pagopa.interop.probing.eservice.operations.annotations.validator;
+
+import it.pagopa.interop.probing.eservice.operations.annotations.ValidateStringArraySize;
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class StringArrayValidator implements ConstraintValidator<ValidateStringArraySize, String[]> {
+
+	int maxSize;
+
+	@Override
+	public void initialize(ValidateStringArraySize constraintAnnotation) {
+		maxSize = constraintAnnotation.maxSize();
+	}
+
+	@Override
+	public boolean isValid(String[] array, ConstraintValidatorContext context) {
+		if (Objects.nonNull(array)) {
```

Comment:
Use Stream if possible

Reply:
We already have a task in the backlog for this change. Changes will be done in IESP-152 task.